NtopNG fails to compile against latest nDPI

Reference ticket  https://github.com/ntop/ntopng/issues/7026 for ntop.  nDPI information:

~~~
commit ada4fe4aa8f88300cfc0dbe6ee965975274b1c40 (HEAD -> dev, origin/dev, origin/HEAD)
Author: Ivan Nardi <12729895+IvanNardi@users.noreply.github.com>
Date:   Tue Dec 6 17:41:58 2022 +0100
~~~

updated with git pull.  Configured with:
~~~
./configure --prefix=/opt/ntopng/ndpi
~~~

Configured and installed just fine.

Enclosed.  Also, I decided to completed delete the old /opt/ntopng/ndpi directory, and after making and installing I see this when I try and configure ntopng:
~~~
configure: error: ERROR: Unable to find custom nDPI static library in /opt/ntopng/ndpi/lib
~~~
it appears that nothing actually gets installed in /opt/ntopng/ndpi/lib:
~~~
[11:01:42 me@gateway:~/nobackup/build/ntopng$] ls -l /opt/ntopng/ndpi/lib/
total 4
drwxr-xr-x 2 root root 4096 Dec  8 10:50 pkgconfig
[11:02:08 me@gateway:~/nobackup/build/ntopng$]
~~~

config for ntopng:
~~~
 ./configure --prefix=/opt/ntopng --with-ndpi-static-lib=/opt/ntopng/ndpi/lib --with-ndpi-includes=/opt/ntopng/ndpi/include/ndpi
~~~



~~~
This file contains any messages produced by compilers while
running configure, to aid debugging if configure makes a mistake.

It was created by libndpi configure 4.5.0, which was
generated by GNU Autoconf 2.69.  Invocation command line was

  $ ./configure --prefix=/opt/ntopng/ndpi

## --------- ##
## Platform. ##
## --------- ##

hostname = gateway
uname -m = x86_64
uname -r = 5.4.0-135-generic
uname -s = Linux
uname -v = #152-Ubuntu SMP Wed Nov 23 20:19:22 UTC 2022

/usr/bin/uname -p = unknown
/bin/uname -X     = unknown

/bin/arch              = unknown
/usr/bin/arch -k       = unknown
/usr/convex/getsysinfo = unknown
/usr/bin/hostinfo      = unknown
/bin/machine           = unknown
/usr/bin/oslevel       = unknown
/bin/universe          = unknown

PATH: /usr/local/sbin
PATH: /usr/local/bin
PATH: /usr/sbin
PATH: /usr/bin
PATH: /sbin
PATH: /bin
PATH: /usr/games
PATH: /usr/local/games
PATH: /snap/bin
PATH: /usr/lib/jvm/java-8-oracle/bin
PATH: /usr/lib/jvm/java-8-oracle/db/bin
PATH: /usr/lib/jvm/java-8-oracle/jre/bin
PATH: /opt/bin
PATH: /opt/sbin
PATH: /opt/local/bin
PATH: /opt/local/sbin


## ----------- ##
## Core tests. ##
## ----------- ##

configure:2517: checking for a BSD-compatible install
configure:2585: result: /usr/bin/install -c
configure:2596: checking whether build environment is sane
configure:2651: result: yes
configure:2800: checking for a thread-safe mkdir -p
configure:2839: result: /bin/mkdir -p
configure:2846: checking for gawk
configure:2862: found /usr/bin/gawk
configure:2873: result: gawk
configure:2884: checking whether make sets $(MAKE)
configure:2906: result: yes
configure:2935: checking whether make supports nested variables
configure:2952: result: yes
configure:3278: checking build system type
configure:3292: result: x86_64-pc-linux-gnu
configure:3312: checking host system type
configure:3325: result: x86_64-pc-linux-gnu
configure:3366: checking how to print strings
configure:3393: result: printf
configure:3418: checking whether make supports the include directive
configure:3433: make -f confmf.GNU && cat confinc.out
this is the am__doit target
configure:3436: $? = 0
configure:3455: result: yes (GNU style)
configure:3525: checking for gcc
configure:3541: found /usr/bin/gcc
configure:3552: result: gcc
configure:3781: checking for C compiler version
configure:3790: gcc --version >&5
gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

configure:3801: $? = 0
configure:3790: gcc -v >&5
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/9/lto-wrapper
OFFLOAD_TARGET_NAMES=nvptx-none:hsa
OFFLOAD_TARGET_DEFAULT=1
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 9.4.0-1ubuntu1~20.04.1' --with-bugurl=file:///usr/share/doc/gcc-9/README.Bugs --enable-languages=c,ada,c++,go,brig,d,fortran,objc,obj-c++,gm2 --prefix=/usr --with-gcc-major-version-only --program-suffix=-9 --program-prefix=x86_64-linux-gnu- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-plugin --enable-default-pie --with-system-zlib --with-target-system-zlib=auto --enable-objc-gc=auto --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-offload-targets=nvptx-none=/build/gcc-9-Av3uEd/gcc-9-9.4.0/debian/tmp-nvptx/usr,hsa --without-cuda-driver --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 9.4.0 (Ubuntu 9.4.0-1ubuntu1~20.04.1)
configure:3801: $? = 0
configure:3790: gcc -V >&5
gcc: error: unrecognized command line option '-V'
gcc: fatal error: no input files
compilation terminated.
configure:3801: $? = 1
configure:3790: gcc -qversion >&5
gcc: error: unrecognized command line option '-qversion'; did you mean '--version'?
gcc: fatal error: no input files
compilation terminated.
configure:3801: $? = 1
configure:3821: checking whether the C compiler works
configure:3843: gcc    conftest.c  >&5
configure:3847: $? = 0
configure:3895: result: yes
configure:3898: checking for C compiler default output file name
configure:3900: result: a.out
configure:3906: checking for suffix of executables
configure:3913: gcc -o conftest    conftest.c  >&5
configure:3917: $? = 0
configure:3939: result:
configure:3961: checking whether we are cross compiling
configure:3969: gcc -o conftest    conftest.c  >&5
configure:3973: $? = 0
configure:3980: ./conftest
configure:3984: $? = 0
configure:3972: result: no
configure:3977: checking for suffix of object files
configure:3999: gcc -c   conftest.c >&5
configure:4003: $? = 0
configure:4024: result: o
configure:4028: checking whether we are using the GNU C compiler
configure:4047: gcc -c   conftest.c >&5
configure:4047: $? = 0
configure:4056: result: yes
configure:4065: checking whether gcc accepts -g
configure:4085: gcc -c -g  conftest.c >&5
configure:4085: $? = 0
configure:4126: result: yes
configure:4143: checking for gcc option to accept ISO C89
configure:4206: gcc  -c -g -O2  conftest.c >&5
configure:4206: $? = 0
configure:4219: result: none needed
configure:4244: checking whether gcc understands -c and -o together
configure:4266: gcc -c conftest.c -o conftest2.o
configure:4269: $? = 0
configure:4266: gcc -c conftest.c -o conftest2.o
configure:4269: $? = 0
configure:4281: result: yes
configure:4300: checking dependency style of gcc
configure:4411: result: gcc3
configure:4426: checking for a sed that does not truncate output
configure:4490: result: /bin/sed
configure:4508: checking for grep that handles long lines and -e
configure:4566: result: /bin/grep
configure:4571: checking for egrep
configure:4633: result: /bin/grep -E
configure:4638: checking for fgrep
configure:4700: result: /bin/grep -F
configure:4735: checking for ld used by gcc
configure:4802: result: /usr/bin/ld
configure:4809: checking if the linker (/usr/bin/ld) is GNU ld
configure:4824: result: yes
configure:4836: checking for BSD- or MS-compatible name lister (nm)
configure:4890: result: /usr/bin/nm -B
configure:5020: checking the name lister (/usr/bin/nm -B) interface
configure:5027: gcc -c -g -O2  conftest.c >&5
configure:5030: /usr/bin/nm -B "conftest.o"
configure:5033: output
0000000000000000 B some_variable
configure:5034: result: BSD nm
configure:5037: checking whether ln -s works
configure:5041: result: yes
configure:5049: checking the maximum length of command line arguments
configure:5180: result: 1572864
configure:5228: checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format
configure:5268: result: func_convert_file_noop
configure:5275: checking how to convert x86_64-pc-linux-gnu file names to toolchain format
configure:5295: result: func_convert_file_noop
configure:5302: checking for /usr/bin/ld option to reload object files
configure:5309: result: -r
configure:5383: checking for objdump
configure:5399: found /usr/bin/objdump
configure:5410: result: objdump
configure:5442: checking how to recognize dependent libraries
configure:5642: result: pass_all
configure:5727: checking for dlltool
configure:5757: result: no
configure:5787: checking how to associate runtime and link libraries
configure:5814: result: printf %s\n
configure:5875: checking for ar
configure:5891: found /usr/bin/ar
configure:5902: result: ar
configure:5939: checking for archiver @FILE support
configure:5956: gcc -c -g -O2  conftest.c >&5
configure:5956: $? = 0
configure:5959: ar cr libconftest.a @conftest.lst >&5
configure:5962: $? = 0
configure:5967: ar cr libconftest.a @conftest.lst >&5
ar: conftest.o: No such file or directory
configure:5970: $? = 1
configure:5969: result: @
configure:6027: checking for strip
configure:6043: found /usr/bin/strip
configure:6054: result: strip
configure:6126: checking for ranlib
configure:6142: found /usr/bin/ranlib
configure:6153: result: ranlib
configure:6255: checking command to parse /usr/bin/nm -B output from gcc object
configure:6408: gcc -c -g -O2  conftest.c >&5
configure:6411: $? = 0
configure:6415: /usr/bin/nm -B conftest.o | sed -n -e 's/^.*[    ]\([ABCDGIRSTW][ABCDGIRSTW]*\)[         ][      ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p' | sed '/ __gnu_lto/d' > conftest.nm
configure:6481: gcc -o conftest -g -O2   conftest.c conftstm.o >&5
configure:6484: $? = 0
configure:6522: result: ok
configure:6569: checking for sysroot
configure:6599: result: no
configure:6606: checking for a working dd
configure:6644: result: /bin/dd
configure:6648: checking how to truncate binary pipes
configure:6663: result: /bin/dd bs=4096 count=1
configure:6799: gcc -c -g -O2  conftest.c >&5
configure:6802: $? = 0
configure:6992: checking for mt
configure:7008: found /bin/mt
configure:7019: result: mt
configure:7042: checking if mt is a manifest tool
configure:7048: mt '-?'
configure:7056: result: no
configure:7733: checking how to run the C preprocessor
configure:7764: gcc -E  conftest.c
configure:7764: $? = 0
configure:7778: gcc -E  conftest.c
conftest.c:11:10: fatal error: ac_nonexistent.h: No such file or directory
   11 | #include <ac_nonexistent.h>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
configure:7778: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libndpi"
| #define PACKAGE_TARNAME "libndpi"
| #define PACKAGE_VERSION "4.5.0"
| #define PACKAGE_STRING "libndpi 4.5.0"
| #define PACKAGE_BUGREPORT ""
| #define PACKAGE_URL ""
| #define PACKAGE "libndpi"
| #define VERSION "4.5.0"
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:7803: result: gcc -E
configure:7823: gcc -E  conftest.c
configure:7823: $? = 0
configure:7837: gcc -E  conftest.c
conftest.c:11:10: fatal error: ac_nonexistent.h: No such file or directory
   11 | #include <ac_nonexistent.h>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
configure:7837: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libndpi"
| #define PACKAGE_TARNAME "libndpi"
| #define PACKAGE_VERSION "4.5.0"
| #define PACKAGE_STRING "libndpi 4.5.0"
| #define PACKAGE_BUGREPORT ""
| #define PACKAGE_URL ""
| #define PACKAGE "libndpi"
| #define VERSION "4.5.0"
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:7866: checking for ANSI C header files
configure:7886: gcc -c -g -O2  conftest.c >&5
configure:7886: $? = 0
configure:7959: gcc -o conftest -g -O2   conftest.c  >&5
configure:7959: $? = 0
configure:7959: ./conftest
configure:7959: $? = 0
configure:7970: result: yes
configure:7983: checking for sys/types.h
configure:7983: gcc -c -g -O2  conftest.c >&5
configure:7983: $? = 0
configure:7983: result: yes
configure:7983: checking for sys/stat.h
configure:7983: gcc -c -g -O2  conftest.c >&5
configure:7983: $? = 0
configure:7983: result: yes
configure:7983: checking for stdlib.h
configure:7983: gcc -c -g -O2  conftest.c >&5
configure:7983: $? = 0
configure:7983: result: yes
configure:7983: checking for string.h
configure:7983: gcc -c -g -O2  conftest.c >&5
configure:7983: $? = 0
configure:7983: result: yes
configure:7983: checking for memory.h
configure:7983: gcc -c -g -O2  conftest.c >&5
configure:7983: $? = 0
configure:7983: result: yes
configure:7983: checking for strings.h
configure:7983: gcc -c -g -O2  conftest.c >&5
configure:7983: $? = 0
configure:7983: result: yes
configure:7983: checking for inttypes.h
configure:7983: gcc -c -g -O2  conftest.c >&5
configure:7983: $? = 0
configure:7983: result: yes
configure:7983: checking for stdint.h
configure:7983: gcc -c -g -O2  conftest.c >&5
configure:7983: $? = 0
configure:7983: result: yes
configure:7983: checking for unistd.h
configure:7983: gcc -c -g -O2  conftest.c >&5
configure:7983: $? = 0
configure:7983: result: yes
configure:7997: checking for dlfcn.h
configure:7997: gcc -c -g -O2  conftest.c >&5
configure:7997: $? = 0
configure:7997: result: yes
configure:8253: checking for objdir
configure:8268: result: .libs
configure:8532: checking if gcc supports -fno-rtti -fno-exceptions
configure:8550: gcc -c -g -O2  -fno-rtti -fno-exceptions conftest.c >&5
cc1: warning: command line option '-fno-rtti' is valid for C++/D/ObjC++ but not for C
configure:8554: $? = 0
configure:8567: result: no
configure:8931: checking for gcc option to produce PIC
configure:8938: result: -fPIC -DPIC
configure:8946: checking if gcc PIC flag -fPIC -DPIC works
configure:8964: gcc -c -g -O2  -fPIC -DPIC -DPIC conftest.c >&5
configure:8968: $? = 0
configure:8981: result: yes
configure:9010: checking if gcc static flag -static works
configure:9038: result: yes
configure:9053: checking if gcc supports -c -o file.o
configure:9074: gcc -c -g -O2  -o out/conftest2.o conftest.c >&5
configure:9078: $? = 0
configure:9100: result: yes
configure:9108: checking if gcc supports -c -o file.o
configure:9155: result: yes
configure:9188: checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries
configure:10451: result: yes
configure:10488: checking whether -lc should be explicitly linked in
configure:10496: gcc -c -g -O2  conftest.c >&5
configure:10499: $? = 0
configure:10514: gcc -shared  -fPIC -DPIC conftest.o  -v -Wl,-soname -Wl,conftest -o conftest 2\>\&1 \| /bin/grep  -lc  \>/dev/null 2\>\&1
configure:10517: $? = 0
configure:10531: result: no
configure:10691: checking dynamic linker characteristics
configure:11272: gcc -o conftest -g -O2   -Wl,-rpath -Wl,/foo conftest.c  >&5
configure:11272: $? = 0
configure:11521: result: GNU/Linux ld.so
configure:11643: checking how to hardcode library paths into programs
configure:11668: result: immediate
configure:12216: checking whether stripping libraries is possible
configure:12221: result: yes
configure:12256: checking if libtool supports shared libraries
configure:12258: result: yes
configure:12261: checking whether to build shared libraries
configure:12286: result: yes
configure:12289: checking whether to build static libraries
configure:12293: result: yes
configure:9194: checking for cos in -lm
configure:9219: gcc -o conftest -g -O2   conftest.c -lm   >&5
conftest.c:30:6: warning: conflicting types for built-in function 'cos'; expected 'double(double)' [-Wbuiltin-declaration-mismatch]
   30 | char cos ();
      |      ^~~
conftest.c:1:1: note: 'cos' is declared in header '<math.h>'
    1 | /* confdefs.h */
configure:9219: $? = 0
configure:9228: result: yes
configure:9292: checking for pkg-config
configure:9310: found /usr/bin/pkg-config
configure:9322: result: /usr/bin/pkg-config
configure:9347: checking pkg-config is at least version 0.9.0
configure:9350: result: yes
configure:9407: checking for gcc
configure:9434: result: gcc
configure:9663: checking for C compiler version
configure:9672: gcc --version >&5
gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

configure:9683: $? = 0
configure:9672: gcc -v >&5
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/9/lto-wrapper
OFFLOAD_TARGET_NAMES=nvptx-none:hsa
OFFLOAD_TARGET_DEFAULT=1
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 9.4.0-1ubuntu1~20.04.1' --with-bugurl=file:///usr/share/doc/gcc-9/README.Bugs --enable-languages=c,ada,c++,go,brig,d,fortran,objc,obj-c++,gm2 --prefix=/usr --with-gcc-major-version-only --program-suffix=-9 --program-prefix=x86_64-linux-gnu- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-plugin --enable-default-pie --with-system-zlib --with-target-system-zlib=auto --enable-objc-gc=auto --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-offload-targets=nvptx-none=/build/gcc-9-Av3uEd/gcc-9-9.4.0/debian/tmp-nvptx/usr,hsa --without-cuda-driver --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 9.4.0 (Ubuntu 9.4.0-1ubuntu1~20.04.1)
configure:9683: $? = 0
configure:9672: gcc -V >&5
gcc: error: unrecognized command line option '-V'
gcc: fatal error: no input files
compilation terminated.
configure:9683: $? = 1
configure:9672: gcc -qversion >&5
gcc: error: unrecognized command line option '-qversion'; did you mean '--version'?
gcc: fatal error: no input files
compilation terminated.
configure:9683: $? = 1
configure:9687: checking whether we are using the GNU C compiler
configure:9715: result: yes
configure:9724: checking whether gcc accepts -g
configure:9785: result: yes
configure:9802: checking for gcc option to accept ISO C89
configure:9878: result: none needed
configure:9903: checking whether gcc understands -c and -o together
configure:9940: result: yes
configure:9959: checking dependency style of gcc
configure:10070: result: gcc3
configure:13806: checking whether gcc and cc understand -c and -o together
configure:13837: gcc -c conftest.c -o conftest2.o >&5
configure:13841: $? = 0
configure:13847: gcc -c conftest.c -o conftest2.o >&5
configure:13851: $? = 0
configure:13862: cc -c conftest.c >&5
configure:13866: $? = 0
configure:13874: cc -c conftest.c -o conftest2.o >&5
configure:13878: $? = 0
configure:13884: cc -c conftest.c -o conftest2.o >&5
configure:13888: $? = 0
configure:13906: result: yes
configure:13974: checking for g++
configure:13990: found /usr/bin/g++
configure:14001: result: g++
configure:14028: checking for C++ compiler version
configure:14037: g++ --version >&5
g++ (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

configure:14048: $? = 0
configure:14037: g++ -v >&5
Using built-in specs.
COLLECT_GCC=g++
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/9/lto-wrapper
OFFLOAD_TARGET_NAMES=nvptx-none:hsa
OFFLOAD_TARGET_DEFAULT=1
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 9.4.0-1ubuntu1~20.04.1' --with-bugurl=file:///usr/share/doc/gcc-9/README.Bugs --enable-languages=c,ada,c++,go,brig,d,fortran,objc,obj-c++,gm2 --prefix=/usr --with-gcc-major-version-only --program-suffix=-9 --program-prefix=x86_64-linux-gnu- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-plugin --enable-default-pie --with-system-zlib --with-target-system-zlib=auto --enable-objc-gc=auto --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-offload-targets=nvptx-none=/build/gcc-9-Av3uEd/gcc-9-9.4.0/debian/tmp-nvptx/usr,hsa --without-cuda-driver --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 9.4.0 (Ubuntu 9.4.0-1ubuntu1~20.04.1)
configure:14048: $? = 0
configure:14037: g++ -V >&5
g++: error: unrecognized command line option '-V'
g++: fatal error: no input files
compilation terminated.
configure:14048: $? = 1
configure:14037: g++ -qversion >&5
g++: error: unrecognized command line option '-qversion'; did you mean '--version'?
g++: fatal error: no input files
compilation terminated.
configure:14048: $? = 1
configure:14052: checking whether we are using the GNU C++ compiler
configure:14071: g++ -c   conftest.cpp >&5
configure:14071: $? = 0
configure:14080: result: yes
configure:14089: checking whether g++ accepts -g
configure:14109: g++ -c -g  conftest.cpp >&5
configure:14109: $? = 0
configure:14150: result: yes
configure:14175: checking dependency style of g++
configure:14286: result: gcc3
configure:14346: checking for ar
configure:14373: result: ar
configure:14468: checking for JSONC
configure:14475: $PKG_CONFIG --exists --print-errors "json-c"
configure:14478: $? = 0
configure:14492: $PKG_CONFIG --exists --print-errors "json-c"
configure:14495: $? = 0
configure:14535: result: yes
configure:14545: checking for json_object_put in -ljson-c
configure:14570: gcc -o conftest -g -O2   conftest.c -ljson-c   >&5
configure:14570: $? = 0
configure:14579: result: yes
configure:14754: checking whether gcc is Clang
configure:14779: result: no
configure:14841: checking whether pthreads work with "-pthread" and "-lpthread"
configure:14946: gcc -o conftest -g -O2 -pthread   conftest.c -lpthread  >&5
configure:14946: $? = 0
configure:14955: result: yes
configure:15072: checking for joinable pthread attribute
configure:15090: gcc -o conftest -g -O2 -pthread   conftest.c -lpthread  >&5
configure:15090: $? = 0
configure:15098: result: PTHREAD_CREATE_JOINABLE
configure:15112: checking whether more special flags are required for pthreads
configure:15125: result: no
configure:15133: checking for PTHREAD_PRIO_INHERIT
configure:15150: gcc -o conftest -g -O2 -pthread   conftest.c -lpthread  >&5
configure:15150: $? = 0
configure:15159: result: yes
configure:15354: checking netinet/in.h usability
configure:15354: gcc -c -g -O2  conftest.c >&5
configure:15354: $? = 0
configure:15354: result: yes
configure:15354: checking netinet/in.h presence
configure:15354: gcc -E  conftest.c
configure:15354: $? = 0
configure:15354: result: yes
configure:15354: checking for netinet/in.h
configure:15354: result: yes
configure:15354: checking for stdint.h
configure:15354: result: yes
configure:15354: checking for stdlib.h
configure:15354: result: yes
configure:15354: checking for string.h
configure:15354: result: yes
configure:15354: checking for unistd.h
configure:15354: result: yes
configure:15354: checking math.h usability
configure:15354: gcc -c -g -O2  conftest.c >&5
configure:15354: $? = 0
configure:15354: result: yes
configure:15354: checking math.h presence
configure:15354: gcc -E  conftest.c
configure:15354: $? = 0
configure:15354: result: yes
configure:15354: checking for math.h
configure:15354: result: yes
configure:15354: checking float.h usability
configure:15354: gcc -c -g -O2  conftest.c >&5
configure:15354: $? = 0
configure:15354: result: yes
configure:15354: checking float.h presence
configure:15354: gcc -E  conftest.c
configure:15354: $? = 0
configure:15354: result: yes
configure:15354: checking for float.h
configure:15354: result: yes
configure:15364: checking for sqrt in -lm
configure:15389: gcc -o conftest -g -O2   conftest.c -lm   >&5
conftest.c:45:6: warning: conflicting types for built-in function 'sqrt'; expected 'double(double)' [-Wbuiltin-declaration-mismatch]
   45 | char sqrt ();
      |      ^~~~
conftest.c:1:1: note: 'sqrt' is declared in header '<math.h>'
    1 | /* confdefs.h */
configure:15389: $? = 0
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libndpi"
| #define PACKAGE_TARNAME "libndpi"
| #define PACKAGE_VERSION "4.5.0"
| #define PACKAGE_STRING "libndpi 4.5.0"
| #define PACKAGE_BUGREPORT ""
| #define PACKAGE_URL ""
| #define PACKAGE "libndpi"
| #define VERSION "4.5.0"
| #define STDC_HEADERS 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_MEMORY_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_UNISTD_H 1
| #define HAVE_DLFCN_H 1
| #define LT_OBJDIR ".libs/"
| #define NDPI_MAJOR_RELEASE "4"
| #define NDPI_MINOR_RELEASE "5"
| #define NDPI_PATCH_LEVEL "0"
| #define NDPI_GIT_RELEASE "4.5.0-3953-ada4fe4a"
| #define NDPI_GIT_DATE "Tue Dec 6 17:41:58 2022 +0100"
| #define HAVE_LIBJSON_C 1
| #define HAVE_PTHREAD_PRIO_INHERIT 1
| #define HAVE_PTHREAD 1
| #define HAVE_NETINET_IN_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_UNISTD_H 1
| #define HAVE_MATH_H 1
| #define HAVE_FLOAT_H 1
| /* end confdefs.h.  */
|
| /* Override any GCC internal prototype to avoid an error.
|    Use char because int might match the return type of a GCC
|    builtin and then its argument prototype would still apply.  */
| #ifdef __cplusplus
| extern "C"
| #endif
| char sqrt ();
| int
| main ()
| {
| return sqrt ();
|   ;
|   return 0;
| }
configure:15398: result: no
configure:15411: checking for rrd_fetch_r in -lrrd
configure:15436: gcc -o conftest -g -O2   conftest.c -lrrd   >&5
configure:15436: $? = 0
configure:15445: result: yes
configure:15466: checking DPDK (used by ndpiReader)
configure:15473: result: no (missing /home/me/DPDK)
configure:15484: checking for numa_available in -lnuma
configure:15509: gcc -o conftest -g -O2   conftest.c -lnuma   >&5
/usr/bin/ld: cannot find -lnuma
collect2: error: ld returned 1 exit status
configure:15509: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "libndpi"
| #define PACKAGE_TARNAME "libndpi"
| #define PACKAGE_VERSION "4.5.0"
| #define PACKAGE_STRING "libndpi 4.5.0"
| #define PACKAGE_BUGREPORT ""
| #define PACKAGE_URL ""
| #define PACKAGE "libndpi"
| #define VERSION "4.5.0"
| #define STDC_HEADERS 1
| #define HAVE_SYS_TYPES_H 1
| #define HAVE_SYS_STAT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_MEMORY_H 1
| #define HAVE_STRINGS_H 1
| #define HAVE_INTTYPES_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_UNISTD_H 1
| #define HAVE_DLFCN_H 1
| #define LT_OBJDIR ".libs/"
| #define NDPI_MAJOR_RELEASE "4"
| #define NDPI_MINOR_RELEASE "5"
| #define NDPI_PATCH_LEVEL "0"
| #define NDPI_GIT_RELEASE "4.5.0-3953-ada4fe4a"
| #define NDPI_GIT_DATE "Tue Dec 6 17:41:58 2022 +0100"
| #define HAVE_LIBJSON_C 1
| #define HAVE_PTHREAD_PRIO_INHERIT 1
| #define HAVE_PTHREAD 1
| #define HAVE_NETINET_IN_H 1
| #define HAVE_STDINT_H 1
| #define HAVE_STDLIB_H 1
| #define HAVE_STRING_H 1
| #define HAVE_UNISTD_H 1
| #define HAVE_MATH_H 1
| #define HAVE_FLOAT_H 1
| #define HAVE_RRDTOOL "1"
| /* end confdefs.h.  */
|
| /* Override any GCC internal prototype to avoid an error.
|    Use char because int might match the return type of a GCC
|    builtin and then its argument prototype would still apply.  */
| #ifdef __cplusplus
| extern "C"
| #endif
| char numa_available ();
| int
| main ()
| {
| return numa_available ();
|   ;
|   return 0;
| }
configure:15518: result: no
configure:15529: checking PF_RING nBPF (/home/me/nobackup/build/nDPI/../PF_RING/userland/nbpf)
configure:15544: result: no
configure:15894: checking for pcap_open_live in -lpcap
configure:15919: gcc -o conftest -g -O2   conftest.c -lpcap   >&5
configure:15919: $? = 0
configure:15928: result: yes
configure:16592: checking for pthread_setaffinity_np in -lpthread
configure:16617: gcc -o conftest -g -O2   conftest.c -lpthread   >&5
configure:16617: $? = 0
configure:16626: result: yes
configure:17016: checking that generated files are newer than configure
configure:17022: result: done
configure:17073: creating ./config.status

## ---------------------- ##
## Running config.status. ##
## ---------------------- ##

This file was extended by libndpi config.status 4.5.0, which was
generated by GNU Autoconf 2.69.  Invocation command line was

  CONFIG_FILES    =
  CONFIG_HEADERS  =
  CONFIG_LINKS    =
  CONFIG_COMMANDS =
  $ ./config.status

on gateway

config.status:1228: creating Makefile
config.status:1228: creating example/Makefile
config.status:1228: creating example/Makefile.dpdk
config.status:1228: creating tests/Makefile
config.status:1228: creating tests/unit/Makefile
config.status:1228: creating tests/performance/Makefile
config.status:1228: creating tests/dga/Makefile
config.status:1228: creating rrdtool/Makefile
config.status:1228: creating influxdb/Makefile
config.status:1228: creating libndpi.pc
config.status:1228: creating src/include/ndpi_define.h
config.status:1228: creating src/lib/Makefile
config.status:1228: creating fuzz/Makefile
config.status:1228: creating doc/Doxyfile.cfg
config.status:1228: creating tests/do.sh
config.status:1228: creating src/include/ndpi_config.h
config.status:1409: src/include/ndpi_config.h is unchanged
config.status:1457: executing depfiles commands
config.status:1534: cd fuzz       && sed -e '/# am--include-marker/d' Makefile         | make -f - am--depfiles
make: Nothing to be done for 'am--depfiles'.
config.status:1539: $? = 0
config.status:1457: executing libtool commands

## ---------------- ##
## Cache variables. ##
## ---------------- ##

ac_cv_build=x86_64-pc-linux-gnu
ac_cv_c_compiler_gnu=yes
ac_cv_cxx_compiler_gnu=yes
ac_cv_env_CCC_set=
ac_cv_env_CCC_value=
ac_cv_env_CC_set=
ac_cv_env_CC_value=
ac_cv_env_CFLAGS_set=
ac_cv_env_CFLAGS_value=
ac_cv_env_CPPFLAGS_set=
ac_cv_env_CPPFLAGS_value=
ac_cv_env_CPP_set=
ac_cv_env_CPP_value=
ac_cv_env_CXXCPP_set=
ac_cv_env_CXXCPP_value=
ac_cv_env_CXXFLAGS_set=
ac_cv_env_CXXFLAGS_value=
ac_cv_env_CXX_set=
ac_cv_env_CXX_value=
ac_cv_env_JSONC_CFLAGS_set=
ac_cv_env_JSONC_CFLAGS_value=
ac_cv_env_JSONC_LIBS_set=
ac_cv_env_JSONC_LIBS_value=
ac_cv_env_LDFLAGS_set=
ac_cv_env_LDFLAGS_value=
ac_cv_env_LIBS_set=
ac_cv_env_LIBS_value=
ac_cv_env_LT_SYS_LIBRARY_PATH_set=
ac_cv_env_LT_SYS_LIBRARY_PATH_value=
ac_cv_env_PCAP_CFLAGS_set=
ac_cv_env_PCAP_CFLAGS_value=
ac_cv_env_PCAP_LIBS_set=
ac_cv_env_PCAP_LIBS_value=
ac_cv_env_PKG_CONFIG_LIBDIR_set=
ac_cv_env_PKG_CONFIG_LIBDIR_value=
ac_cv_env_PKG_CONFIG_PATH_set=
ac_cv_env_PKG_CONFIG_PATH_value=
ac_cv_env_PKG_CONFIG_set=
ac_cv_env_PKG_CONFIG_value=
ac_cv_env_PROFILER_CFLAGS_set=
ac_cv_env_PROFILER_CFLAGS_value=
ac_cv_env_PROFILER_LIBS_set=
ac_cv_env_PROFILER_LIBS_value=
ac_cv_env_TCMALLOC_CFLAGS_set=
ac_cv_env_TCMALLOC_CFLAGS_value=
ac_cv_env_TCMALLOC_LIBS_set=
ac_cv_env_TCMALLOC_LIBS_value=
ac_cv_env_build_alias_set=
ac_cv_env_build_alias_value=
ac_cv_env_host_alias_set=
ac_cv_env_host_alias_value=
ac_cv_env_target_alias_set=
ac_cv_env_target_alias_value=
ac_cv_header_dlfcn_h=yes
ac_cv_header_float_h=yes
ac_cv_header_inttypes_h=yes
ac_cv_header_math_h=yes
ac_cv_header_memory_h=yes
ac_cv_header_netinet_in_h=yes
ac_cv_header_stdc=yes
ac_cv_header_stdint_h=yes
ac_cv_header_stdlib_h=yes
ac_cv_header_string_h=yes
ac_cv_header_strings_h=yes
ac_cv_header_sys_stat_h=yes
ac_cv_header_sys_types_h=yes
ac_cv_header_unistd_h=yes
ac_cv_host=x86_64-pc-linux-gnu
ac_cv_lib_json_c_json_object_put=yes
ac_cv_lib_m_cos=yes
ac_cv_lib_m_sqrt=no
ac_cv_lib_numa_numa_available=no
ac_cv_lib_pcap_pcap_open_live=yes
ac_cv_lib_pthread_pthread_setaffinity_np=yes
ac_cv_lib_rrd_rrd_fetch_r=yes
ac_cv_objext=o
ac_cv_path_EGREP='/bin/grep -E'
ac_cv_path_FGREP='/bin/grep -F'
ac_cv_path_GREP=/bin/grep
ac_cv_path_SED=/bin/sed
ac_cv_path_ac_pt_PKG_CONFIG=/usr/bin/pkg-config
ac_cv_path_install='/usr/bin/install -c'
ac_cv_path_lt_DD=/bin/dd
ac_cv_path_mkdir=/bin/mkdir
ac_cv_prog_AWK=gawk
ac_cv_prog_CPP='gcc -E'
ac_cv_prog_ac_ct_AR=ar
ac_cv_prog_ac_ct_CC=gcc
ac_cv_prog_ac_ct_CXX=g++
ac_cv_prog_ac_ct_MANIFEST_TOOL=mt
ac_cv_prog_ac_ct_OBJDUMP=objdump
ac_cv_prog_ac_ct_RANLIB=ranlib
ac_cv_prog_ac_ct_STRIP=strip
ac_cv_prog_cc_c89=
ac_cv_prog_cc_g=yes
ac_cv_prog_cc_gcc_c_o=yes
ac_cv_prog_cxx_g=yes
ac_cv_prog_make_make_set=yes
am_cv_CC_dependencies_compiler_type=gcc3
am_cv_CXX_dependencies_compiler_type=gcc3
am_cv_make_support_nested_variables=yes
am_cv_prog_cc_c_o=yes
ax_cv_PTHREAD_CLANG=no
ax_cv_PTHREAD_JOINABLE_ATTR=PTHREAD_CREATE_JOINABLE
ax_cv_PTHREAD_PRIO_INHERIT=yes
ax_cv_PTHREAD_SPECIAL_FLAGS=no
lt_cv_ar_at_file=@
lt_cv_archive_cmds_need_lc=no
lt_cv_deplibs_check_method=pass_all
lt_cv_file_magic_cmd='$MAGIC_CMD'
lt_cv_file_magic_test_file=
lt_cv_ld_reload_flag=-r
lt_cv_nm_interface='BSD nm'
lt_cv_objdir=.libs
lt_cv_path_LD=/usr/bin/ld
lt_cv_path_NM='/usr/bin/nm -B'
lt_cv_path_mainfest_tool=no
lt_cv_prog_compiler_c_o=yes
lt_cv_prog_compiler_pic='-fPIC -DPIC'
lt_cv_prog_compiler_pic_works=yes
lt_cv_prog_compiler_rtti_exceptions=no
lt_cv_prog_compiler_static_works=yes
lt_cv_prog_gnu_ld=yes
lt_cv_sharedlib_from_linklib_cmd='printf %s\n'
lt_cv_shlibpath_overrides_runpath=yes
lt_cv_sys_global_symbol_pipe='sed -n -e '\''s/^.*[       ]\([ABCDGIRSTW][ABCDGIRSTW]*\)[         ][      ]*\([_A-Za-z][_A-Za-z0-9]*\)$/\1 \2 \2/p'\'' | sed '\''/ __gnu_lto/d'\'''
lt_cv_sys_global_symbol_to_c_name_address='sed -n -e '\''s/^: \(.*\) .*$/  {"\1", (void *) 0},/p'\'' -e '\''s/^[ABCDGIRSTW][ABCDGIRSTW]* .* \(.*\)$/  {"\1", (void *) \&\1},/p'\'''
lt_cv_sys_global_symbol_to_c_name_address_lib_prefix='sed -n -e '\''s/^: \(.*\) .*$/  {"\1", (void *) 0},/p'\'' -e '\''s/^[ABCDGIRSTW][ABCDGIRSTW]* .* \(lib.*\)$/  {"\1", (void *) \&\1},/p'\'' -e '\''s/^[ABCDGIRSTW][ABCDGIRSTW]* .* \(.*\)$/  {"lib\1", (void *) \&\1},/p'\'''
lt_cv_sys_global_symbol_to_cdecl='sed -n -e '\''s/^T .* \(.*\)$/extern int \1();/p'\'' -e '\''s/^[ABCDGIRSTW][ABCDGIRSTW]* .* \(.*\)$/extern char \1;/p'\'''
lt_cv_sys_global_symbol_to_import=
lt_cv_sys_max_cmd_len=1572864
lt_cv_to_host_file_cmd=func_convert_file_noop
lt_cv_to_tool_file_cmd=func_convert_file_noop
lt_cv_truncate_bin='/bin/dd bs=4096 count=1'
pkg_cv_JSONC_CFLAGS=-I/usr/include/json-c
pkg_cv_JSONC_LIBS=-ljson-c

## ----------------- ##
## Output variables. ##
## ----------------- ##

ACLOCAL='${SHELL} /home/me/nobackup/build/nDPI/missing aclocal-1.16'
ADDITIONAL_INCS=''
ADDITIONAL_LIBS='-lm'
AMDEPBACKSLASH='\'
AMDEP_FALSE='#'
AMDEP_TRUE=''
AMTAR='$${TAR-tar}'
AM_BACKSLASH='\'
AM_DEFAULT_V='$(AM_DEFAULT_VERBOSITY)'
AM_DEFAULT_VERBOSITY='1'
AM_V='$(V)'
AR='ar'
AUTOCONF='${SHELL} /home/me/nobackup/build/nDPI/missing autoconf'
AUTOHEADER='${SHELL} /home/me/nobackup/build/nDPI/missing autoheader'
AUTOMAKE='${SHELL} /home/me/nobackup/build/nDPI/missing automake-1.16'
AWK='gawk'
BUILD_FUZZTARGETS='0'
BUILD_FUZZTARGETS_FALSE=''
BUILD_FUZZTARGETS_TRUE='#'
BUILD_MINGW=''
BUILD_MINGW_X64=''
BUILD_UNITTESTS_FALSE='#'
BUILD_UNITTESTS_TRUE=''
CC='gcc'
CCDEPMODE='depmode=gcc3'
CFLAGS='-g -O2'
CPP='gcc -E'
CPPFLAGS=''
CUSTOM_NDPI=''
CXX='g++'
CXXCPP=''
CXXDEPMODE='depmode=gcc3'
CXXFLAGS='-g -O2'
CYGPATH_W='echo'
DEFS='-DHAVE_CONFIG_H'
DEPDIR='.deps'
DISABLE_NPCAP='0'
DLLTOOL='false'
DL_LIB=''
DPDK_TARGET=''
DSYMUTIL=''
DUMPBIN=''
ECHO_C=''
ECHO_N='-n'
ECHO_T=''
EGREP='/bin/grep -E'
EXEEXT=''
EXE_SUFFIX=''
EXTRA_TARGETS='example tests tests/dga tests/unit rrdtool'
FGREP='/bin/grep -F'
GIT_RELEASE='4.5.0-3953-ada4fe4a'
GPROF_CFLAGS=''
GPROF_ENABLED='0'
GPROF_LIBS=''
GREP='/bin/grep'
HANDLE_TLS_SIGS='//'
HAS_FUZZLDFLAGS_FALSE=''
HAS_FUZZLDFLAGS_TRUE='#'
HAVE_PTHREAD_SETAFFINITY_NP=''
INSTALL_DATA='${INSTALL} -m 644'
INSTALL_PROGRAM='${INSTALL}'
INSTALL_SCRIPT='${INSTALL}'
INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
JSONC_CFLAGS='-I/usr/include/json-c'
JSONC_LIBS='-ljson-c'
LD='/usr/bin/ld -m elf_x86_64'
LDFLAGS=''
LIBM='-lm'
LIBOBJS=''
LIBRRD='-lrrd'
LIBS=''
LIBTOOL='$(SHELL) $(top_builddir)/libtool'
LIB_FUZZING_ENGINE=''
LIPO=''
LN_S='ln -s'
LTLIBOBJS=''
LT_SYS_LIBRARY_PATH=''
MAKEINFO='${SHELL} /home/me/nobackup/build/nDPI/missing makeinfo'
MANIFEST_TOOL=':'
MKDIR_P='/bin/mkdir -p'
NBPF_ENABLED='0'
NDPI_API_VERSION='7147'
NDPI_CFLAGS='-W -Wall -Wno-unused-parameter -Wno-unused-function -Wno-address-of-packed-member  -O2'
NDPI_LDFLAGS=''
NDPI_MAJOR='4'
NDPI_MINOR='5'
NDPI_PATCH='0'
NDPI_VERSION_SHORT='4.5.0'
NM='/usr/bin/nm -B'
NMEDIT=''
OBJDUMP='objdump'
OBJEXT='o'
OTOOL64=''
OTOOL=''
PACKAGE='libndpi'
PACKAGE_BUGREPORT=''
PACKAGE_NAME='libndpi'
PACKAGE_STRING='libndpi 4.5.0'
PACKAGE_TARNAME='libndpi'
PACKAGE_URL=''
PACKAGE_VERSION='4.5.0'
PATH_SEPARATOR=':'
PCAP_CFLAGS=''
PCAP_INC=''
PCAP_LIB='-lpcap'
PCAP_LIBS=''
PCRE_ENABLED='0'
PKG_CONFIG='/usr/bin/pkg-config'
PKG_CONFIG_LIBDIR=''
PKG_CONFIG_PATH=''
PROFILER_CFLAGS=''
PROFILER_LIBS=''
PTHREAD_CC='gcc'
PTHREAD_CFLAGS='-pthread'
PTHREAD_CXX='g++'
PTHREAD_LIBS='-lpthread'
RANLIB='ranlib'
SED='/bin/sed'
SET_MAKE=''
SHELL='/bin/bash'
STRIP='strip'
TCMALLOC_CFLAGS=''
TCMALLOC_LIBS=''
USE_HOST_LIBGCRYPT='0'
VERSION='4.5.0'
ac_ct_AR='ar'
ac_ct_CC='gcc'
ac_ct_CXX='g++'
ac_ct_DUMPBIN=''
am__EXEEXT_FALSE=''
am__EXEEXT_TRUE='#'
am__fastdepCC_FALSE='#'
am__fastdepCC_TRUE=''
am__fastdepCXX_FALSE='#'
am__fastdepCXX_TRUE=''
am__include='include'
am__isrc=''
am__leading_dot='.'
am__nodep='_no'
am__quote=''
am__tar='$${TAR-tar} chof - "$$tardir"'
am__untar='$${TAR-tar} xf -'
ax_pthread_config=''
bindir='${exec_prefix}/bin'
build='x86_64-pc-linux-gnu'
build_alias=''
build_cpu='x86_64'
build_os='linux-gnu'
build_vendor='pc'
datadir='${datarootdir}'
datarootdir='${prefix}/share'
docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
dvidir='${docdir}'
exec_prefix='${prefix}'
host='x86_64-pc-linux-gnu'
host_alias=''
host_cpu='x86_64'
host_os='linux-gnu'
host_vendor='pc'
htmldir='${docdir}'
includedir='${prefix}/include'
infodir='${datarootdir}/info'
install_sh='${SHELL} /home/me/nobackup/build/nDPI/install-sh'
libdir='${exec_prefix}/lib'
libexecdir='${exec_prefix}/libexec'
localedir='${datarootdir}/locale'
localstatedir='${prefix}/var'
mandir='${datarootdir}/man'
mkdir_p='$(MKDIR_P)'
oldincludedir='/usr/include'
pdfdir='${docdir}'
prefix='/opt/ntopng/ndpi'
program_transform_name='s,x,x,'
psdir='${docdir}'
runstatedir='${localstatedir}/run'
sbindir='${exec_prefix}/sbin'
sharedstatedir='${prefix}/com'
sysconfdir='${prefix}/etc'
target_alias=''

## ----------- ##
## confdefs.h. ##
## ----------- ##

/* confdefs.h */
#define PACKAGE_NAME "libndpi"
#define PACKAGE_TARNAME "libndpi"
#define PACKAGE_VERSION "4.5.0"
#define PACKAGE_STRING "libndpi 4.5.0"
#define PACKAGE_BUGREPORT ""
#define PACKAGE_URL ""
#define PACKAGE "libndpi"
#define VERSION "4.5.0"
#define STDC_HEADERS 1
#define HAVE_SYS_TYPES_H 1
#define HAVE_SYS_STAT_H 1
#define HAVE_STDLIB_H 1
#define HAVE_STRING_H 1
#define HAVE_MEMORY_H 1
#define HAVE_STRINGS_H 1
#define HAVE_INTTYPES_H 1
#define HAVE_STDINT_H 1
#define HAVE_UNISTD_H 1
#define HAVE_DLFCN_H 1
#define LT_OBJDIR ".libs/"
#define NDPI_MAJOR_RELEASE "4"
#define NDPI_MINOR_RELEASE "5"
#define NDPI_PATCH_LEVEL "0"
#define NDPI_GIT_RELEASE "4.5.0-3953-ada4fe4a"
#define NDPI_GIT_DATE "Tue Dec 6 17:41:58 2022 +0100"
#define HAVE_LIBJSON_C 1
#define HAVE_PTHREAD_PRIO_INHERIT 1
#define HAVE_PTHREAD 1
#define HAVE_NETINET_IN_H 1
#define HAVE_STDINT_H 1
#define HAVE_STDLIB_H 1
#define HAVE_STRING_H 1
#define HAVE_UNISTD_H 1
#define HAVE_MATH_H 1
#define HAVE_FLOAT_H 1
#define HAVE_RRDTOOL "1"
#define HAVE_PTHREAD_SETAFFINITY_NP 1

configure: exit 0
~~~

--- a/src/lib/Makefile.in
+++ b/src/lib/Makefile.in
@@ -11,7 +11,7 @@ RANLIB = @RANLIB@
 #
 # Installation directories
 #
-prefix     = @prefix@
+PREFIX     = @prefix@
 libdir     = @libdir@
 includedir = @includedir@/ndpi
 ifneq ($(OS),Windows_NT)
@@ -91,9 +91,9 @@ cppcheck:
 	cppcheck --template='{file}:{line}:{severity}:{message}' --quiet --enable=all --force -I ../include *.c protocols/*.c
 
 install: $(NDPI_LIBS)
-	mkdir -p $(DESTDIR)$(libdir)
-	cp $(NDPI_LIBS) $(DESTDIR)$(libdir)/
-	cp -P $(NDPI_LIB_SHARED_BASE) $(DESTDIR)$(libdir)/
-	cp -P $(NDPI_LIB_SHARED_BASE).$(NDPI_VERSION_MAJOR) $(DESTDIR)$(libdir)/
-	mkdir -p $(DESTDIR)$(includedir)
-	cp ../include/*.h $(DESTDIR)$(includedir)
+	mkdir -p $(DESTDIR)$(PREFIX)$(libdir)
+	cp $(NDPI_LIBS) $(DESTDIR)$(PREFIX)$(libdir)/
+	cp -P $(NDPI_LIB_SHARED_BASE) $(DESTDIR)$(PREFIX)$(libdir)/
+	cp -P $(NDPI_LIB_SHARED_BASE).$(NDPI_VERSION_MAJOR) $(DESTDIR)$(PREFIX)$(libdir)/
+	mkdir -p $(DESTDIR)$(PREFIX)$(includedir)
+	cp ../include/*.h $(DESTDIR)$(PREFIX)$(includedir)

